src/main/java/com/cburch/logisim/std/arith/DividerHdlGeneratorFactory.java

```diff
@@ -52,20 +52,20 @@ public LineBuffer getModuleFunctionality(Netlist TheNetlist, AttributeSet attrs)
             .pair("calcBits", CALC_BITS_STRING);
 
     if (Hdl.isVhdl()) {
-      contents.add("""
-          s_extended_dividend({{calcBits}}-1 DOWNTO {{nrOfBits}}) <= Upper;
-          s_extended_dividend({{nrOfBits}}-1 DOWNTO 0) <= INP_A;
-          s_div_result <= std_logic_vector(unsigned(s_extended_dividend) / unsigned(INP_B))
-                             WHEN {{unsigned}} = 1 ELSE
-                          std_logic_vector(signed(s_extended_dividend) / signed(INP_B));
-          s_mod_result <= std_logic_vector(unsigned(s_extended_dividend) mod unsigned(INP_B))
-                             WHEN {{unsigned}} = 1 ELSE
-                          std_logic_vector(signed(s_extended_dividend) mod signed(INP_B));
-          Quotient  <= s_div_result({{nrOfBits}}-1 DOWNTO 0);
-          Remainder <= s_mod_result({{nrOfBits}}-1 DOWNTO 0);
+      contents.empty().addVhdlKeywords().add("""
+          s_extendedDividend({{calcBits}}-1 {{downto}} {{nrOfBits}}) <= upper;
+          s_extendedDividend({{nrOfBits}}-1 {{downto}} 0) <= inputA;
+          s_divResult <= std_logic_vector(unsigned(s_extendedDividend) / unsigned(inputB))
+                             {{when}} {{unsigned}} = 1 {{else}}
```

Comment:
wouldn't it be more readable to move `{{else}}` to next line and align with `{{when}}`? (multiple)

Reply:
This is a coding convention question, I'll leave it as is for the moment until we decide on the VHDL coding convention.